### ISSUE TITLE
Bitrunning doesn't work

### ISSUE THREAD

**Issue by Borisvanmemes** [tgstation/tgstation#84238]
Reporting client version: 515.1637

## Issue Summary

Netpods simply give you a "Out of bandwidth!' message whenever you get in them and do nothing else.

## Round ID:
[230959](https://scrubby.melonmesa.com/round/230959)

## Testmerges:
- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Cult Vs. Heretic: 7 Months Later Edition](https://www.github.com/tgstation/tgstation/pull/82877)
- [Bitrunning: Combat domain](https://www.github.com/tgstation/tgstation/pull/84196)

## Reproduction:
Load a domain with the console, get in a pod, don't get sent to the domain. I was unable to properly confirm whether or not the domain was actually properly loading in in the game world.

**Comment by Sadboysuss** [comment]
If the avatar that the netpod is linked to is unconscious or there isn't an avatar linked to it - it will say out of bandwidth:
```
if(isnull(current_avatar) || current_avatar.stat != CONSCIOUS) // We need a viable avatar
		var/obj/structure/hololadder/wayout = server.generate_hololadder()
		if(isnull(wayout))
			balloon_alert(neo, "out of bandwidth!")
			return
```

**Comment by Rhials** [comment]
I'm willing to wager this is related to #84196 if this is a real bug. Paging @jlsnow301 for input.

**Comment by Borisvanmemes** [comment]
> If the avatar that the netpod is linked to is unconscious or there isn't an avatar linked to it - it will say out of bandwidth:
> 
> ```
> if(isnull(current_avatar) || current_avatar.stat != CONSCIOUS) // We need a viable avatar
> 		var/obj/structure/hololadder/wayout = server.generate_hololadder()
> 		if(isnull(wayout))
> 			balloon_alert(neo, "out of bandwidth!")
> 			return
> ```
> 
> Have you tried reloading the domain?

That was tried yeah, and it didn't work. Neither did de and reconning the pods, de and reconning the server, moving the entire setup, etc etc.

**Comment by jlsnow301** [comment]
Do you know which domain?

There might have been a previous instance that didn't clean up properly

**Comment by Borisvanmemes** [comment]
> Do you know which domain?
> 
> There might have been a previous instance that didn't clean up properly

While I'm not 100% certain of this, I'm fairly certain the first domain that was attempted to be loaded in was the beach bar one. After that the gondola asteroid was loaded in, and it didn't make a difference.

**Comment by jlsnow301** [comment]
> > ```
> > if(isnull(current_avatar) || current_avatar.stat != CONSCIOUS) // We need a viable avatar
> > 		var/obj/structure/hololadder/wayout = server.generate_hololadder()
> > 		if(isnull(wayout))
> > 			balloon_alert(neo, "out of bandwidth!")
> > 			return
> > ```

This is NOT the code that's on master rn btw. So perhaps, with the recent merge of #84125, there was some sort of conflict

**Comment by jlsnow301** [comment]
Fixed this issue in [623ce71](https://github.com/tgstation/tgstation/pull/84196/commits/623ce71d55b9f99d40cbfde6dcef00512fd424db)
